devfile.yaml

```diff
@@ -2,6 +2,12 @@
 apiVersion: 1.0.0
 metadata:
   generateName: che-docs-
+  
+projects:
+  - name: che-docs
+    source:
+      type: git
+      location: "https://github.com/eclipse/che-docs.git"
```

Comment:
assuming you don't want to lock this to a specific branch, then +1

Reply:
@MichalMaler , @themr0c: is it okay to open master branch of che-docs project by default?

Reply:
The project is intentionally absent from the devfile to make it working on forks.

The devfile is not supposed to work when taken out of context.

Reply:
Got it.